spec/factories/cohorts.rb

```diff
@@ -2,19 +2,10 @@
 
 FactoryBot.define do
   factory :cohort do
-    to_create do |instance|
-      if (existing = Cohort.find_by(start_year: instance.start_year))
-        instance.attributes = existing.attributes
-        instance.instance_variable_set("@new_record", false)
-      else
-        instance.save!
-      end
-    end
-
-    start_year { Faker::Number.unique.between(from: 2021, to: 2100) }
+    start_year { Faker::Number.unique.between(from: Date.current.year + (Date.current.month >= 9 ? 1 : 2), to: 2100) }
 
     trait :current do
-      start_year { 2021 }
```

Comment:
That's the point in September when the cohort will change and this will break `Cohort.current`. 

Changing this to 2022 will break most since due to `Cohort.current`'s implementation

Reply:
But that will be in a controlled manner with some planned work. With this change come the 1st of September the whole pipeline will be blocked. And other people wanting to ship their changes will be blocked by this. they will simply change it back to `2021` to resolve it after figuring out what went wrong to unblock the pipe